src/groovy/au/org/emii/aatams/data/DevelopmentDataInitialiser.groovy

```diff
@@ -465,7 +465,6 @@ class DevelopmentDataInitialiser extends AbstractDataInitialiser
         WKTReader reader = new WKTReader();
 
         Point location = (Point)reader.read("POINT(30.1234 30.1234)")
-        location.setSRID(4326)
```

Comment:
Deliberate - this was causing an error in one of the SQL `GROUP BY` clauses, trying to group geometries with different SRIDs (i.e. 4326 and null)